src/main/java/core/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.service;
+
+import core.storage.Storage;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String CHAR_FOR_SPLIT = ",";
+
+    private Storage storage;
+
+    public ReportServiceImpl(Storage storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public String create() {
+        return storage.getAllData()
```

Comment:
or we can pass data to the method:
```suggestion
    public String create(Map<String, Integer> data) {
        return data
```